packages/components/src/__future__/Tag/Tag/_docs/Tag.mdx

```diff
@@ -52,3 +52,11 @@ in LTR, and on the right of `children` in RTL. For a list of icons, see [Icons](
 Allows classnames to be attached to the outermost element of Tag.
 
 <Canvas of={TagStories.ClassNameOverride} />
+
+### Migrating from legacy tag
```

Comment:
```suggestion
### Migrating from legacy Tag
```

Reply:
I'd just link to the other doc instead of listing things in here.

Reply:
```suggestion
### Migrating to Future Tag
```
Probably better to rephrase it like this, so we remove the Legacy